apps/opik-backend/src/main/java/com/comet/opik/api/sorting/SpanSortingFactory.java

```diff
@@ -41,6 +42,7 @@ public List<String> getSortableFields() {
                 END_TIME,
                 DURATION,
                 USAGE,
+                USAGE_DYNAMIC,
```

Comment:
The frontend proably needs to know the difference between USAGE_DYNAMIC and USAGE